src/components/Base/Base.tsx

```diff
@@ -1,39 +1,26 @@
-import * as React from 'react'
+import React, { FC, Ref } from 'react'
 import styled, { css } from 'styled-components'
 
-type Radius = 's' | 'm'
-
-interface Props {
-  radius?: Radius
-  children?: React.ReactNode
+type BaseProps = {
+  radius?: 's' | 'm'
   className?: string
+  ref?: Ref<HTMLDivElement>
```

Comment:
Is this prepared to access the DOM information of the Base component?
If so, I think you should use [Ref forwarding](https://ja.reactjs.org/docs/forwarding-refs.html).